Syntaxes/Swift.tmLanguage

```diff
@@ -4533,9 +4637,165 @@
 								</dict>
 							</array>
 						</dict>
+						<dict>
+							<key>begin</key>
+							<string>(##+)"</string>
+							<key>beginCaptures</key>
+							<dict>
+								<key>0</key>
+								<dict>
+									<key>name</key>
+									<string>punctuation.definition.string.begin.raw.swift</string>
+								</dict>
+							</dict>
+							<key>comment</key>
+							<string>SE-0168: raw string literals (more than one #, grammar limitations prevent us from supporting escapes)</string>
+							<key>end</key>
+							<string>"\1(#*)</string>
+							<key>endCaptures</key>
+							<dict>
+								<key>0</key>
+								<dict>
+									<key>name</key>
+									<string>punctuation.definition.string.end.raw.swift</string>
+								</dict>
+								<key>1</key>
+								<dict>
+									<key>name</key>
+									<string>invalid.illegal.extra-closing-delimiter.swift</string>
+								</dict>
+							</dict>
+							<key>name</key>
+							<string>string.quoted.double.single-line.raw.swift</string>
+							<key>patterns</key>
+							<array>
+								<dict>
+									<key>match</key>
+									<string>\r|\n</string>
```

Comment:
Does clang actually treat a single `\r` as a newline character (and thus disallow these in raw strings)?

Reply:
Good question; yes, I just tested and a literal \r or \n are both rejected in both raw and regular strings.

![](https://user-images.githubusercontent.com/14237/63633282-c8086500-c5fa-11e9-9155-61fa1e58d2c1.png)